src/QMidiOut.cpp

```diff
@@ -54,6 +54,21 @@ void QMidiOut::pitchWheel(int voice, int value)
 	sendMsg(msg);
 }
 
+void QMidiOut::channelAftertouch(int voice, int value)
+{
+    qint32 msg = 0xD0 + voice;
+    msg |= value << 8;
+    sendMsg(msg);
```

Comment:
All of this (and the below) needs to be indented by one tab, and not four spaces.